pantheon-bundle/frontend/package.json

```diff
@@ -22,6 +22,7 @@
     "clean": "rimraf dist"
   },
   "devDependencies": {
+    "@testing-library/react": "^10.1.0",
```

Comment:
The latest version is 11.2.3. I wonder why we don't use the latest?

Reply:
thanks for catching. this was the version installed by default. Updated the pr